src/utils/isPlainObject.js

```diff
@@ -0,0 +1,42 @@
+/*!
+ * is-plain-object <https://github.com/jonschlinkert/is-plain-object>
+ *
+ * Copyright (c) 2014-2017, Jon Schlinkert.
+ * Released under the MIT License.
+ */
+
+import isObject from "./isObject";
+
+const isObjectObject = o => {
+  return (
+    isObject(o) === true &&
+    Object.prototype.toString.call(o) === "[object Object]"
+  );
+};
+
+/**
+ * Determines if the value is an object created by the object
+ * constructor.
+ * @param value
+ * @returns {boolean}
+ */
+export default value => {
+  if (isObjectObject(value) === false) return false;
+
+  // If has modified constructor
+  const ctor = value.constructor;
+  if (typeof ctor !== "function") return false;
```

Comment:
Would it be better to check that "ctor === Object" ?
![image](https://user-images.githubusercontent.com/7866685/64729609-b815cf80-d49a-11e9-80cc-8ebf69176dc5.png)

Reply:
And then you wouldn't have to do the other two check I think

Reply:
This code is copied from the [`is-plain-object` package](https://github.com/jonschlinkert/is-plain-object) (see the header at the top of the file) but modified only to re-use our own `isObject` utility. I'd rather not modify the code because what's here is very well battle-tested (depended on by 3+ million github projects, installed from npm 11+ million times weekly).